randovania/games/dread/json_data/Burenia.txt

```diff
@@ -1391,6 +1393,8 @@ Extra - asset_id: collision_camera_010
 
 > Alcove Across Grapple Block; Heals? False
   * Layers: default
+  > Door to Energy Recharge South
+      Speed Booster and After Burenia - Prepare Speedboost in Save Station and After Burenia - Main Hub Bottom Blob and Speedbooster Conservation (Beginner)
```

Comment:
So the reason i put conservation as an alternative is that if you dont have gravity, you cannot retry. I would make this Gravity or conservation, but i do agree this one can be beginner. The other one you have to pull the block then spark within the timer while the enemy is there, so that's why that one is intermediate

Reply:
I added using shinespark with only gravity from above the screw attack blocks, so I don't think there's any reason for this one to have an alternative with gravity.